resources/images/filters.go

```diff
@@ -33,6 +35,53 @@ func (*Filters) Overlay(src ImageSource, x, y interface{}) gift.Filter {
 	}
 }
 
+// Text creates a filter that draws text with the given options.
+func (*Filters) Text(text string, options ...interface{}) gift.Filter {
+	tf := textFilter{
+		text:        text,
+		color:       "#ffffff",
+		size:        20,
+		x:           10,
+		y:           10,
+		linespacing: 2,
+	}
+
+	var opt map[string]interface{}
+	if len(options) > 0 {
+		var err error
+		opt, err = maps.ToStringMapE(options[0])
```

Comment:
I suggest you just use `maps.MustToParamsAndPrepare`; this makes all keys into lowercase, which makes the compare below simpler.